ui/src/app/app-routing.tsx

```diff
@@ -246,4 +98,16 @@ export class AppRouting extends ReactWrapperBase {
     AnalyticsTracker.Registration.SignIn();
     this.signInService.signIn();
   }
+
+  onSignOut(): void {
```

Comment:
Were you able to verify that the inactivity expiration sign out still works after this change?

Reply:
It does!